linux_pipeline/Jenkinsfile_upstream_kernel_customer

```diff
@@ -58,6 +53,10 @@ pipeline {
     USE_KERNEL_FOLDER_PREFIX = 'True'
     THREAD_NUMBER = 'x3'
     AZURE_BOOT_LARGE_FLAVOR = 'Standard_E64_v3'
+    KERNEL_GIT_BRANCH = 'master'
+    LISA_TEST_XML = 'kernel_pipeline_bvt.xml'
+    KERNEL_DEBUG = 'False'
+    ENABLED_STAGES = 'build_artifacts, publish_temp_artifacts, boot_test, publish_artifacts, validation_functional, validation_functional_hyperv, publish_azure_vhd, validation_functional_azure, validation_perf_azure'
```

Comment:
do you want to have the boot test with small or large flavor?
now, it boot test only with Azure small flavor

Reply:
move this lines at the beginning of the environment, as they are first to be taken into consideration and easier to spot there.

Reply:
I am not sure, but as long as it works, I will leave it regarding Azure Small Flavor. 
Code line moved to the top of env def.